webpack.config.js

```diff
@@ -223,16 +225,21 @@ let appConfig = {
         include: [staticPrefix],
         exclude: /(vendor|node_modules|dist)/,
         use: [
-          {
-            loader: 'babel-loader',
-            options: babelOptions,
-          },
-          {
-            loader: 'ts-loader',
-            options: {
-              transpileOnly: false,
-            },
-          },
+          ...(!IS_CI
```

Comment:
I kind of dislike this style prop spreading, I think it's a tad hard to read) but 🤷‍♂

I'd maybe prefer to have variables for like,  `tsLoaderConfig` and `bableLoaderConfig` and then down here have

```jsx
...(!IS_CI ? [bableLoaderConfig] : [tsLoaderConfig]),
```

Reply:
hmmm the spread doesn't look like it's even necessary either